ldms/python/ldmsd/ldmsd_controller

```diff
@@ -292,18 +292,18 @@ class LdmsdCmdParser(cmd.Cmd):
         Load a plugin at the Aggregator/Producer
         Parameters:
         name=   The plugin to load
-        inst=     The configuration name for the plugin (default = <name>)
+        plugin= The configuration name for the plugin (default = <name>)
         """
         arg = self.handle_args('load', arg)
         if arg:
             name = arg['name']
-            if arg['inst']:
-                inst = arg['inst']
+            if arg['plugin']:
+                plugin = arg['plugin']
             else:
```

Comment:
We should drop this "else" and just leave plugin as None. Otherwise this is triply redudantly handling what the server already handles. This stops us from checking the ldmsd_communicator.py is handling a missing "plugin=" correctly, which is in turn stopping us for checking that the server is doing it right.